test_conformance/api/test_mem_object_info.cpp

```diff
@@ -217,6 +217,9 @@ int test_get_buffer_info( cl_device_id deviceID, cl_context context, cl_command_
             // Create a buffer object to test against.
             bufferObject = clCreateBuffer( context, bufferFlags[ i ], addressAlign * 4, NULL, &error );
             test_error( error, "Unable to create buffer to test with" );
+            void *ptr;
+            TEST_MEM_OBJECT_PARAM(bufferObject, CL_MEM_HOST_PTR, ptr, NULL,
+                                  "host pointer", "%p", void *)
```

Comment:
This is fine, but as-written it is only testing two scenarios: the USE_HOST_PTR case (tested above, ~line 177), and this case when no other flags are passed.  Would it be better to refactor this part of the test slightly, so this query gets tested in more cases as part of "Perform buffer object queries" below, vs. being performed here when the buffer is created?